R/validation.R

```diff
@@ -111,16 +111,16 @@ validate_outcomes_are_numeric <- function(outcomes) {
   check <- check_outcomes_are_numeric(outcomes)
 
   if (!check$ok) {
-    bad_cols <- glue::single_quote(names(check$bad_classes))
-    bad_printable_classes <- map(check$bad_classes, glue_quote_collapse)
-    bad_msg <- glue::glue("{bad_cols}: {bad_printable_classes}")
-    bad_msg <- glue::glue_collapse(bad_msg, sep = "\n")
-
-    glubort(
-      "All outcomes must be numeric, but the following are not:",
-      "\n",
-      "{bad_msg}"
-    )
```

Comment:
This pattern produces a very readable output so I decided to preserve it, even if means formatting and gluing prior to `cli_abort()`.